src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+/* html {
+  margin: 0;
+  padding: 0;
+} */
+
+.search-form {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+  margin: 20px 0;
+}
+
+.search-form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form:focus-within {
+  border-radius: 3px;
+}
+
+.search-form-big {
+  height: 68px;
+}
+
+.search-form-small {
+  height: 38px;
+  padding-bottom: 2px;
+}
+
+.image-search-big {
+  width: 19px;
+  height: 19px;
+  margin: 24px 17px 25px 25px;
+}
+
+.image-search-small {
+  width: 11px;
+  height: 11px;
+  margin: 14px 9px 15px 12px;
+}
+
+.input {
+  border: none;
+  outline: none;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
move in to `html` and inherit here

Reply:
Ok

Reply:
I don't understand. The input element is not direct child of html element. So, how can I inherit from it?